client/my-sites/upgrades/checkout/privacy-protection.jsx

```diff
@@ -30,33 +30,20 @@ module.exports = React.createClass( {
 		this.props.onDialogClose();
 	},
 
-	getDomainRegistrations: function() {
-		return cartItems.getDomainRegistrations( this.props.cart );
-	},
-
-	getFirstDomainToRegister: function() {
-		const domainRegistration = this.getDomainRegistrations().shift();
-
-		return domainRegistration.meta;
-	},
-
 	hasDomainPartOfPlan: function() {
 		const cart = this.props.cart;
 		return cart.has_bundle_credit || cartItems.hasPlan( cart );
 	},
 
-	getNumberOfDomainRegistrations: function() {
-		return this.getDomainRegistrations().length;
-	},
-
 	getPrivacyProtectionCost: function() {
 		const products = this.props.productsList.get();
-
 		return products.private_whois.cost_display;
 	},
 
 	render: function() {
-		const numberOfDomainRegistrations = this.getNumberOfDomainRegistrations(),
+		const domainRegistrations = cartItems.getDomainRegistrations( this.props.cart ),
+			numberOfDomainRegistrations = domainRegistrations.length,
+			firstDomainToRegister = domainRegistrations.shift(),
```

Comment:
Hmm, when you shift, `domainRegistrations` is smaller for 1 element. There are still checks that use `domainRegistrations` below, and might be wrong if you do it like this. :)